docs/community/README.md

```diff
@@ -0,0 +1,97 @@
+# How to Get Involved
+
+<!-- TODO: what is community, what are the touchpoints -->
+
+## Questions
+
+For the fastest response, you can ask questions on the [`#general`, `#serving`
+or `#eventing` channels of the Knative Slack](https://slack.knative.dev/).
```

Comment:
Should we add `#functions` to this list?

Reply:
Done.